app/CMakeLists.txt

```diff
@@ -506,10 +508,23 @@ if (LNX)
   target_link_libraries(MerginMaps PUBLIC QGIS::Core)
 endif ()
 
+find_library(
```

Comment:
Would be good to add some explanation comments on why this is needed and what it does